README.md

```diff
@@ -1,3 +1,82 @@
+Plotly, being more sophisticated is used as data visualization tool  for creating elaborate plots more efficiently. 
 
-This was a learning-oriented project done by the devolepers to learn more about git, flask, dash, plotly, deployment,etc and to get the practical experience handling a project. The whole project was done with the data from COVID INDIA API. The site is deployed at https://coviviz.herokuapp.com/.
+
+
+
+## Acknowledgements
+
+ - [COVID INDIA API](https://github.com/covid19india/api)
+  
+
+
+  
+## Demo
+ 
+![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/86158829/136935766-a61f3ca4-ce1d-4fb0-8179-91f13134753f.gif)
+
+  
+
+## Documentation
+This project aims to create a covid webapp.It is a "learning by doing"experience which provided us the practical experience in handling of git,flask,plotly,dash,deployment etc.This project was done with the data from COVID INDIA API.This covid webapp provides the visualizations of vaccination status in the states of India along with the statewise Death rate,Tested Cases,Confirmed Cases,Recovered Cases,Deceased Cases.Thus making  an efficient way of comparison.
+
+  -Statewise Deathrate is plotted in bar chart,vaccination status in pie chart with labels partially_vaccinated,fully vaccinated and not vaccinated.
+  
+  -Clicking on a particular state's bar from the bar chart changes the adjacent pie chart and the vaccination status of that particular state will be shown there.
+  
+  -Placing cursor on a single bar shows the state name and death rate of corresponding state.
+  
+  -Clicking on a legend item of the pie chart makes that item disappear (You can use this feature to tell total vs partially  vaccinated).
+  
+  -Placing the cursor on any particular case of Vaccination Pie chart shows it's label,number of respective case and it's   percentage.
+  
+  -In the state comparison tab, selecting no state will result in the line chart of India.
+  
+  -In the state comparison tab, the Search option is available.
+  
+  -Cases like Tested,Confirmed,Recovered,Deceased are shown in line chart.Each state is shown in different colour .
+  
+  -Multiple states can be viewed altoghther which gives the user the better understanding of case rate in each state.
+  
+  -Clicking on the legend item of any particular selected state,makes the line chart of that state disapper(With this feature we can easily differentiate the lines of other states).
+  
+  -Placing the cursor on any point of line graph shows the state,date and number of respective cases reported.
+  
+## Screenshots
+
+![covidwebapp](https://user-images.githubusercontent.com/86158829/136670586-05271c6a-27fd-40f1-adec-17026fc62bd8.png)
+
+![coviviz](https://user-images.githubusercontent.com/86158829/136839294-3d6c8912-ed34-4d00-b2a3-0afdb152fd43.png)
+
+## Deployment
+
+To see the deployed project, follow the link
+
+```bash
+  https://coviviz.herokuapp.com/.
```

Comment:
It is https://coviviz.herokuapp.com/ , not https://coviviz.herokuapp.com/. .
Please Take care of the "." symbol at the end

Reply:
Done